### DISCUSSION TITLE
Help with install please

### DISCUSSION THREAD

**Discussion by Phunknista05** [Budibase/budibase#2227] [Help] [answered]
Hello all,

I went through steps to download budibase via Digital Ocean, this is the message I get in my shell
Starting services, this may take a moment - first time this may take a few minutes to download images.
root_redis-service_1 is up-to-date
root_couchdb-service_1 is up-to-date
root_watchtower-service_1 is up-to-date
root_minio-service_1 is up-to-date
Starting root_couch-init_1 ... done
bbworker is up-to-date
bbapps is up-to-date
root_proxy-service_1 is up-to-date
Services started, please go to http://localhost:10000 for next steps.
Once I click the link it does not work, any suggestions?

**Comment by mjashanks** [comment, marked as answer]
Hey there. Your Digital Ocean droplet should have a public IP address.

Your budibase should be available on `https://public-ip-address:10000`

It should look a bit like `https://99.88.77.66:10000`

**Comment by Phunknista05** [comment]
It worked!  Thank you Michael!

On Tue, Aug 3, 2021 at 2:06 AM Michael Shanks ***@***.***>
wrote:

> Hey there. Your Digital Ocean droplet should have a public IP address.
>
> Your budibase should be available on https://public-ip-address:10000
>
> It should look a bit like https://99.88.77.66:10000
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/2227#discussioncomment-1123369>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVBNXRXFV2JQLU3W5IIVJDTT26BN3ANCNFSM5BODQDDA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
*RaeDawn Long *
*Chief Technology Officer*

*(502) 855-1927*
***@***.*** ***@***.***>*
*www.linkedin.com/in/dawnlong/ <http://www.linkedin.com/in/dawnlong/>*
